examples/js/exporters/GLTFExporter.js

```diff
@@ -1907,3 +1900,196 @@ THREE.GLTFExporter.prototype = {
 	}
 
 };
+
+THREE.GLTFExporter.Utils = {
+
+	insertKeyframe: function ( track, time ) {
+
+		var tolerance = 0.001; // 1ms
```

Comment:
Can you explain the logic behind this number or is it one just smaller enough to be used as epsilon for the comparison? I'm thinking about the timers precisions changes with the meltdown/spectre and not sure if it has something to do with that value.

Reply:
Nothing that fancy, just an epsilon small enough to be less than the difference between frames (e.g. at 24-30 fps) and large enough to ignore floating point differences between frame values. :)

Reply:
Cool thanks!